### ISSUE TITLE
Add Playback Follows Cursor Checkbox

### ISSUE THREAD

**Issue by Oleksiy-Yakovenko** [DeaDBeeF-Player/deadbeef#450]
Original [issue 521](https://code.google.com/p/ddb/issues/detail?id=521) created by Alexey-Yakovenko on 2012-04-19T10:50:40.000Z:

Under the Playback menu, a &quot;Playback Follows Cursor&quot; checkbox would be nice. What this does is allows one to highlight a song while another one is playing, and when the current song is finished, it will then jump to the highlighted song.

**Comment by abielefeldt** [comment]
I found this issue through a Google search as this feature is not currently implemented in Deadbeef, and it would be extremely useful to have since I'm used to having this functionality in foobar2000 and love being able to "set up" the next song/album/playlist to play when the current song is finished.

I see that the issue has been closed but the comments have been completely garbled so it's unclear why. Is this a "won't fix" issue?
